### PR TITLE
Hide CF accordeon when areaName is not ea

### PR DESCRIPTION
**Pull request by cazapatamar**
I found and fixed a bug with accordion when it a ea is not selected

### CODE REVIEW

File: src/landscape/LandscapeAccordion.jsx [hunk 1]
```diff
@@ -34,6 +36,8 @@ class LandscapeAccordion extends React.Component {
       <div>
         {(componentsArray.length > 0)
           && componentsArray.map(item => (
+            (!item.label.disabled)
+            && (
```
Comment (erikasv) [inline src/landscape/LandscapeAccordion.jsx:40]: Put these in the same line (see https://github.com/airbnb/javascript/tree/master/react#alignment)

Comment (erikasv) [reply src/landscape/LandscapeAccordion.jsx:40]: Also please fix the two previous lines

File: src/landscape/humanFootprint/CurrentFootprint.jsx [hunk 2]
```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import InfoIcon from '@material-ui/icons/Info';
-import ShortInfo from '../commons/ShortInfo';
-import GraphLoader from '../charts/GraphLoader';
-import matchColor from '../commons/matchColor';
+import ShortInfo from '../../commons/ShortInfo';
+import GraphLoader from '../../charts/GraphLoader';
+import matchColor from '../../commons/matchColor';
+
```
Comment (erikasv) [inline src/landscape/humanFootprint/CurrentFootprint.jsx:7]: Keep only one blank line

File: src/landscape/LandscapeAccordion.jsx (hunk 1 above)
Comment (cazapatamar) [reply src/landscape/LandscapeAccordion.jsx]: Done

File: src/landscape/humanFootprint/CurrentFootprint.jsx (hunk 2 above)
Comment (cazapatamar) [reply src/landscape/humanFootprint/CurrentFootprint.jsx:7]: Done

**Pull request merged**